# Programa 1
vetor = [1, 2, 3, 4, 5]
print (vetor[7]) # Retorna o erro